new_relic_apdex_t/new_relic_apdex_t.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/**
+ * @file
+ * Sets New Relic Apdex T values for newly created multidev environments.
+ */
+
+/**
+ * CHANGE THESE VARIABLES FOR YOUR OWN SITE.
+ */
+// The "t" value (number of seconds) for your server-side apdex.
+// https://docs.newrelic.com/docs/apm/new-relic-apm/apdex/apdex-measuring-user-satisfaction
+$app_apdex_threshold = 0.4;
```

Comment:
@stevector I set aside some time Monday to review in more detail.

Rather than hard-coding Apdex score I we can pull target values from the API with the [health status report](https://docs.newrelic.com/docs/apis/rest-api-v2/application-examples-v2/application-reporting-health-status-v2)

Using the target values from dev for new multidev spinups makes more sense to me than having something hard coded. That way if Apdex is changed in New Relic there is no need to remember to update this script.